liberty-maven-plugin/src/it/basic-runtime-properties-override-it/invoker.properties

```diff
@@ -0,0 +1,2 @@
+# prove the runtime version can be overridden
+invoker.goals.1 = clean install -Dliberty.runtime.groupId="io.openliberty" -Dliberty.runtime.artifactId="openliberty-runtime" -Dliberty.runtime.version=19.0.0.7
```

Comment:
Is there a reason the two new property values have quotes around them and the old `liberty.runtime.version` does not? Are both formats valid for all three props?

Reply:
Also, let's update this version number to something more recent...maybe 21.0.0.3 since we are using 21.0.0.6 and 21.0.0.9 in our builds now.